app/src/main/java/org/p2p/wallet/jupiter/statemanager/SwapStateManager.kt

```diff
@@ -235,6 +233,19 @@ class SwapStateManager(
         }
     }
 
+    private fun validateTokens(tokenA: SwapTokenModel, tokenB: SwapTokenModel): Boolean {
```

Comment:
Since it returns `boolean`, we can name it `areValidTokens`

Reply:
or `isTokensAreValid`